uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/menu/SkyBlockMenu.java

```diff
@@ -482,7 +482,7 @@ private void addMainMenu(Inventory menu, Player player) {
         menuItem = new ItemStack(Material.SAPLING, 1, (short) 3);
         meta4 = menuItem.getItemMeta();
         meta4.setDisplayName("\u00a7a\u00a7l" + tr("Change Island Biome"));
-        lores.add("\u00a7eCurrent Biome: \u00a7b" + islandInfo.getBiome());
+        lores.add (tr("\u00a7eCurrent Biome: \u00a7b" + islandInfo.getBiome()));
```

Comment:
The correct line would be:
lores.add(tr("\u00a7eCurrent Biome: \u00a7b{0}", islandInfo.getBiome()));